app/schemas/action.py

```diff
@@ -67,3 +69,7 @@ class MemberSchema(BaseModel):
     class Config:
         from_attributes = True
 
+
+class Role(StrEnum):
+    Admin = "admin"
+    Member = "member"
```

Comment:
its good plactive to make enum by upper case like:
ADMIN = "admin"

and name class like RoleEnum, but ok